nuxt.config.js

```diff
@@ -1,114 +1,69 @@
+          exclude: /(node_modules)/,
+        });
       }
-    }
-  }
-`);
-
-const mapallInternalEventsToRedirects = (allInternalEvents) => (
-  allInternalEvents
-    .map((event) => [
-      `/events/${event.slug} /${event.nativeLocale}/events/${event.slug} 302 Country=${event.countryCode}`,
-      ...event._allNameLocales
-        .filter(({ locale }) => locale !== 'en')
-        .map(({ locale }) =>
-          `/events/${event.slug} /${locale}/events/${event.slug} 302 Language=${locale}`,
-        ),
-      `/events/${event.slug} /en/events/${event.slug} 302`,
-    ])
-    .flat()
-    .join('\n')
-);
 
-const mapallInternalEventsToRoutes = (allInternalEvents) => (
-  allInternalEvents
-    .map((event) =>
-      event._allNameLocales
-        .map((item) => [
-          `${item.locale}/events/${event.slug}`,
-          ...event.sections.map(section =>
-            section.chapters.map(({ slug }) =>
-              `${item.locale}/events/${event.slug}/${slug}`,
-            ),
-          )
-          .flat(),
-        ])
-        .flat(),
-    )
-    .flat()
-);
-
-const postcss = {
-  plugins: {
-    'postcss-custom-media': {
-      importFrom: 'components/core/index.css',
+      config.module.rules.push({
+        test: /\.glsl$/,
+        loader: 'webpack-glsl-loader',
+      });
+    },
+    extractCSS: true,
+    html: {
+      // disable minify CSS and JS to improve build times
+      // https://www.voorhoede.nl/en/blog/10x-faster-nuxt-builds-on-netlify/
+      minify: {
+        collapseBooleanAttributes: true,
+        decodeEntities: true,
+        minifyCSS: false,
+        minifyJS: false,
+        processConditionalComments: true,
+        removeEmptyAttributes: true,
+        removeRedundantAttributes: true,
+        trimCustomFragments: true,
+        useShortDoctype: true,
+      },
+    },
+    postcss: {
+      plugins: {
+        'postcss-custom-media': {
+          importFrom: 'components/core/index.css',
+        },
+        'postcss-calc': {},
+        'postcss-custom-properties': {},
+      },
     },
-    'postcss-calc': {},
-    'postcss-custom-properties': {},
-  },
-};
-
-const routerBase = {
-  router: {
-    base: '/',
   },
-};
-
-const env = {
-  // Allow to choose a baseurl (should only be used during generate)
-  baseUrl: process.env.BASE_URL || 'http://localhost:3000',
-  DATO_API_TOKEN: process.env.DATO_API_TOKEN ,
-};
-
-module.exports = {
   css: [
     '~/components/core/index.css',
     'typeface-roboto',
   ],
+  env: {
+    DATO_API_TOKEN: process.env.DATO_API_TOKEN ,
+  },
+  generate: {
+    exclude: [
+      /^\/events\/.+/,
+      /^\/narratives\/undefined/,
+    ],
+  },
   head: {
     title: 'World Water Atlas',
     meta: [
       { charset: 'utf-8' },
       { name: 'viewport', content: 'width=device-width, initial-scale=1' },
       { hid: 'description', name: 'description', content: 'World Water Atlas' },
-      { 'http-equiv': 'x-ua-compatible', content: 'ie=edge' },
```

Comment:
Since we don't need IE8 & IE9 support, drop it: https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do